common/src/main/java/com/netflix/conductor/common/run/Workflow.java

```diff
@@ -72,6 +74,8 @@ public boolean isSuccessful(){
 	private String reRunFromWorkflowId;
 	
 	private String reasonForIncompletion;
+
+	private String failedTaskId;
```

Comment:
Capture referenceTaskName instead of taskId.

Reply:
Changed this to a `refereceTaskName`